### ISSUE TITLE
Double quote in CREATE TRIGGER code.

### ISSUE THREAD

**Issue by just-doit** [darold/ora2pg#787]
Hi,

it seems I've the same issue as mentioned in #751.
I've already downloaded the master branch from git and tried an export.
Although I'm not 100% sure, the install of the master branch version succeeded, since I had v20.0 before and both seem to report `Ora2Pg v20.0` when doing a `ora2pg -v`.

 Any oracle trigger having a WHEN clause is exported with a ".

Oracle trigger:
```
CREATE OR REPLACE TRIGGER tr_a2p_like_new
   BEFORE UPDATE OF id_account, id_person
   ON account2person
   FOR EACH ROW
   WHEN(   NVL(new.id_account, '-') != NVL(old.id_account, 'x')
        OR NVL(new.id_person, '-') != NVL(old.id_person, 'x'))
BEGIN

```
Converted code:
```
CREATE TRIGGER "tr_a2p_like_new
   before update of id_account, id_person
   on account2person
   for each row"
	WHEN (   coalesce(new.id_account, '-') != coalesce(old.id_account, 'x')
        OR coalesce(new.id_person, '-') != coalesce(old.id_person, 'x'))
	EXECUTE PROCEDURE trigger_fct_tr_a2p_like_new();
```

As you can see, " are inserted into the code so that the parts after the actual name are still interpreted as the name of the trigger.

**Comment by just-doit** [comment]
I think the problem is in line 4633 in file Ora2Pg.om https://github.com/darold/ora2pg/blob/8fb438dde348e3d6cf351d218f52591deca8c2a4/lib/Ora2Pg.pm#L4633

The current statement quotes the entire `DESCRIPTION` of a trigger not just the name.

**Comment by darold** [comment]
Good catch, this should be:
```
$sql_output .= "CREATE TRIGGER " . $self->quote_object_name($trig->[0]) . "\n";
```
can you apply the change and let us know if that's really fix the issue?

**Comment by just-doit** [comment]
I think, If I would just replace the index it would correctly use the trigger name as name (quoted), but the next part added to the statement would be the when clause. The trigger_type, trigger_event and table_name would be missing.

**Comment by darold** [comment]
hum right this is more complicated. I'm sorry but I don't have time to work on Ora2Pg, I will see at end of the week if I can find more time to work on it.

**Comment by just-doit** [comment]
Ok, I'll see If I'm able to fix that on my own. :-) I'll let you known how far I got.

**Comment by darold** [comment]
Fixed by commit 05e47ad
